core/tests/protractor_desktop/voiceoverUploadFeatures.js

```diff
@@ -121,7 +121,6 @@ describe('Voiceover upload features', function() {
       await users.login(TEST_EMAIL);
       await creatorDashboardPage.get();
       await creatorDashboardPage.editExploration(EXPLORATION_TITLE);
-      await explorationEditorMainTab.exitTutorial();
```

Comment:
Why is this not required here and below?
Perhaps we can expect that the tutorial is not open?

Reply:
Welcome Modal appears when the newly created user tries to log in for the very first time. It doesn't on the next login attempts. I tested this in multiple suites.